style.css

```diff
@@ -129,29 +145,63 @@ a {
 }
 
 .divChild2:hover {
-  background: #3f5681;
+  background: white;
   border-color: white;
   border-radius: 10px;
 }
 
 .inputs {
+  font-size: 100%;
   text-align: center;
+  cursor: pointer;
 }
 
-#tempo {
+#tempoSlider {
   margin: 5px 0 5px 0;
+  width: 75%;
+  cursor: pointer;
 }
 
 #startStopBut {
   padding: 15px;

```

Comment:
Something we should talk about in the future are measurement units in CSS. :-) In the meantime, lots of designers work on a 8px grid, so it's generally good to try to size items in multiples of 8px (2px and 4px are fine for things that require that level of grain).

Reply:
OK, let's incorporate some design principles into our next meeting (or whenever you see fit)!